web-components/src/components/modal/modal.stories.tsx

```diff
@@ -323,13 +322,3 @@ export const createSellOrderModal = (): JSX.Element => (
     onSubmit={async () => alert('submit')}
   />
 );
-
```

Comment:
moved to web-registry, so no more need for story

Reply:
Done!